math/eigen/include/algebra/math/impl/eigen_transform3.hpp

```diff
@@ -81,6 +79,8 @@ struct transform3 {
   ALGEBRA_HOST_DEVICE
   transform3(const vector3 &t, const vector3 &x, const vector3 &y,
              const vector3 &z, bool get_inverse = true) {
+    _data.setIdentity();
```

Comment:
I think you also need to set `_data_inv` to identity matrix

Reply:
Ah, right, I keep forgetting that ```get_inverse``` could be ```false```. At that point, accessing ```_data_inv``` should not be permitted actually, regardless if it is the identity matrix or garbage

Reply:
Done